src/js/common/utils/lodash-replacements/get.js

```diff
@@ -0,0 +1,26 @@
+import deconstructPath from './deconstructPath';
+
+/**
+ * Gets a the value at the end of the path.
+ *
+ * @param {Object} object
+ * @param {Array|string} path
+ * @param {*} [defaultValue]
+ * @return {*}
+ */
+export default function get(path, object, defaultValue) {
```

Comment:
I forgot that I had a naive version of this here: https://github.com/department-of-veterans-affairs/vets-website/blob/rjsf-refactor-formapp/src/js/common/schemaform/formState.js#L13

If that helps any.

Reply:
Ahh, very nice. Yeah, you're way is cleaner; I'll adapt.
![image](https://user-images.githubusercontent.com/12970166/34019073-ee4dceda-e0e1-11e7-863b-0072ae0dbdfd.png)